README.md

```diff
@@ -144,3 +144,21 @@ The [raml-module-builder](https://github.com/folio-org/raml-module-builder) fram
 Other [modules](https://dev.folio.org/source-code/#server-side).
 
 Other FOLIO Developer documentation is at [dev.folio.org](https://dev.folio.org/)
+
+### Environment variables
+This module uses S3 storage for files. AWS S3 and Minio Server are supported for files storage.
+It is also necessary to specify variable S3_IS_AWS to determine if AWS S3 is used as files storage. By default,
+this variable is `false` and means that MinIO server is used as storage.
+This value should be `true` if AWS S3 is used.
+
+| Name                    | Default value          | Description                                |
+|:------------------------|:-----------------------|:-------------------------------------------|
+| S3_URL                  | http://127.0.0.1:9000/ | S3 url                                     |
+| S3_REGION               | -                      | S3 region                                  |
+| S3_BUCKET               | -                      | S3 bucket                                  |
+| S3_ACCESS_KEY_ID        | -                      | S3 access key                              |
+| S3_SECRET_ACCESS_KEY    | -                      | S3 secret key                              |
+| S3_IS_AWS               | false                  | Specify if AWS S3 is used as files storage |
+| MY_UNIVERSITY_VALUE     | -                      | The tenant definition value                |
```

Comment:
not sure that this var should be documented as it's system and dynamic, there is no reason to set up it outside

Reply:
fixed